function_template_test.go

```diff
@@ -64,6 +66,36 @@ func TestFunctionTemplateGetFunction(t *testing.T) {
 	}
 }
 
+func TestFunctionCallbackInfoThis(t *testing.T) {
+	t.Parallel()
+
+	iso, _ := v8go.NewIsolate()
+	defer iso.Dispose()
+
+	foo, _ := v8go.NewObjectTemplate(iso)
+	foo.Set("name", "foobar")
+
+	var this *v8go.Object
+	barfn, _ := v8go.NewFunctionTemplate(iso, func(info *v8go.FunctionCallbackInfo) *v8go.Value {
+		this = info.This()
+		return nil
+	})
+	foo.Set("bar", barfn)
+
+	global, _ := v8go.NewObjectTemplate(iso)
+	global.Set("foo", foo)
+
+	ctx, _ := v8go.NewContext(iso, global)
+	defer ctx.Close()
+
+	ctx.RunScript("foo.bar()", "")
+
+	v, _ := this.Get("name")
+	if v.String() != "foobar" {
+		t.Errorf("expected this.name to be foobar, but got %q", v)
+	}
```

Comment:
Looks like this test would be simpler if we exposed [v8::Value::SameValue](https://v8.github.io/api/head/classv8_1_1Value.html#a7c90dc5947b8b99b9089a020fff6d86d) so we could test this with `if foo.SameValue(this)` without the need for "name" in this test.